utils/swift_build_support/swift_build_support/products/product_builder.py

```diff
@@ -0,0 +1,27 @@
+# swift_build_support/product_builders/product_builder.py -------*- python -*-
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
```

Comment:
2019

Reply:
I don’t know why I remember changing that. It must have been in a parallel dimension. I will fix it everywhere.